modules/auxiliary/scanner/smtp/smtp_relay.rb

```diff
@@ -84,6 +84,11 @@ def do_test_relay(testnumber, mailfrom, mailto)
 
       res = raw_send_recv("EHLO X\r\n")
       vprint_status("#{res.inspect}")
+      # check if the EHLO is actually supported. In case it's not, try the HELO command instead
+      if res.inspect =~ /^"5\d\d/
```

Comment:
I assume this would be

```suggestion
      if res.inspect =~ /^5\d\d/
```

Reply:
oh, i see - because you're using `.inspect`

Probably just this then:

```
if "#{res}" =~ /^5\d\d/
```

Reply:
@adfoster-r7 you're right, sorry about that. I fixed this now. Btw I used `res.to_s` since Rubocop prefers that over string interpolation.